runtime/common/src/claims.rs

```diff
@@ -241,7 +267,7 @@ decl_module! {
 		/// - Write: Vesting Vesting, Account, Balance Lock, Total, Claim, Claims Vesting, Signing
 		/// Validate Unsigned: +188.7 µs
 		/// </weight>
-		#[weight = T::DbWeight::get().reads_writes(7, 7) + 270_000_000 + 190_000_000]
+		#[weight = T::WeightInfo::claim().saturating_add(T::WeightInfo::validate_unsigned_claim())]
```

Comment:
at some point wouldn't be better to have all the unsigned and dispatchable logic is the same benchmark ? to avoid counting 2 times some storages ?

Anyway it doesn't really matter here.

Reply:
I could also use whitelisting here, but I think it doesn't matter, and is probably better to keep things simple.

Reply:
took your suggestion!